docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -218,6 +218,48 @@ To remove a traffic light from `TrafficLightPublishPanel`, click the `RESET` but
 
 [Reference video tutorials](https://drive.google.com/file/d/1bs_dX1JJ76qHk-SGvS6YF9gmekkN8fz7/view?usp=sharing)
 
+### Driving across a crosswalk
+
+When driving across a crosswalk, ego vehicle will respond to objects on the crosswalk and the traffic light.
+
+#### Unsignalized crosswalk
+
+To pass through an unsignalized crosswalk, the ego vehicle will first slow down and stop for a safety check. Once safety is confirmed, the vehicle will proceed.
```

Comment:
If there is no obstacle around the cross walk, does the ego really slow down?

Reply:
Resolved by https://github.com/autowarefoundation/autoware-documentation/pull/514/commits/8d3d16d770583c58fc2e14967f0a39d8effcf642